### PR TITLE
feat: added integration service

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/integration/IntegrationService.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  getProviderServices(providerId) {
+    return this.client.send_request({
+      type: 'GET',
+      url: this.client.create_url(
+        `${IntegrationService.URL.PROVIDERS}/${providerId}${IntegrationService.URL.SERVICES}`
+      ),
+    });
+  }
```
Comment (lipis) [inline app/script/integration/IntegrationService.js:57]: one empty line after.. like the rest

File: app/script/integration/IntegrationService.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
```
Comment (gregor) [inline app/script/integration/IntegrationService.js]: 2018

File: app/script/integration/IntegrationService.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  getProvider(providerId) {
+    return this.client.send_request({
+      type: 'GET',
+      url: this.client.create_url(`${IntegrationService.URL.PROVIDERS}/${providerId}`),
+    });
+  }
+
+  getProviderServices(providerId) {
```
Comment (gregor) [inline app/script/integration/IntegrationService.js:50]: Let's keep them alphabetical

File: app/script/main/app.js [hunk 4]
```diff
@@ -125,6 +125,10 @@ z.main.App = class App {
       repositories.user
     );
 
+    repositories.integration = new z.integration.IntegrationRepository(
+      this.service.integration,
+      repositories.conversation
+    );
```
Comment (gregor) [inline app/script/main/app.js]: No line removed or changed just added? You must be adding it twice then. I put them in alphabetical order.

**Pull request merged**